test/plugins/mergeStyles.04.svg

```diff
@@ -0,0 +1,23 @@
+<svg id="test" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
```

Comment:
Add description for each test case

```
description

===

<svg...
```

Reply:
I added descriptions with style similar to git commit messages, like 
`Check whether plugin works with only one style element (no further merging needed, noop).`.